notifications.json

```diff
@@ -0,0 +1,504 @@
+                    "type": "object",
+                    "description": "This parameter specifies the custom key-value pairs of the message's payload.\n\nFor example, with data:{\"score\":\"3x1\"}:\n\nOn iOS, if the message is sent via APNS, it represents the custom data fields. It would be represented as key value dictionary in AppDelegate application:didReceiveRemoteNotification:.\n\nOn Android, this would result in an intent extra named score with the string value 3x1.\n\nThe key should not be a reserved word (\"from\" or any word starting with \"google\" or \"gcm\"). Do not use any of the words defined in this object.\n\nValues in string types are recommended. You have to convert values in objects or other non-string data types (e.g., integers or booleans) to string.",
+                    "properties": {
+                        "icon": {
+                            "type": "string"
+                        },
+                        "sound": {
+                            "type": "string"
+                        },
+                        "color": {
+                            "type": "string"
+                        },
+                        "badge": {
+                            "type": "string"
+                        }
+                    }
+                }
+            },
+            "required": [
+                "title"
+            ]
+        },
+        "message-input": {
+            "title": "Message Input",
+            "type": "object",
+            "properties": {
+                "to": {
+                    "description": "One fiscal code (CF).\n",
+                    "type": "string"
+                },
+                "dry_run": {
+                    "type": "boolean",
+                    "description": "This parameter, when set to true, allows developers to test a request without actually sending a message.\n\nThe default value is false."
+                },
+                "time_to_live": {
+                    "type": "integer",
+                    "description": "This parameter specifies how long (in seconds) the message should be kept in storage if the device is offline. The maximum time to live supported is 4 weeks, and the default value is 4 weeks.",
+                    "minimum": 0
+                },
+                "tags": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "content": {
+                    "$ref": "#/definitions/content"
+                }
+            },
+            "required": [
+                "to",
+                "content"
+            ]
+        },
+        "message-status": {
+            "title": "Message Status",
+            "type": "object",
+            "properties": {
+                "status": {
+                    "type": "string",
+                    "enum": [
+                        "PENDING",
+                        "SENDING",
```

Comment:
I don't like `QUEUED` and `SENDING`, they both leak system abstractions (that notifications can be possibly queued and that they get relayed to a 3rd party system).

Also, we should use an [open-ended list of states](https://zalando.github.io/restful-api-guidelines/compatibility/Compatibility.html#should-used-openended-list-of-values-xextensibleenum-instead-of-enumerations).